src/systemTest/java/com/flightstats/hub/callback/CallbackServer.java

```diff
@@ -0,0 +1,50 @@
+package com.flightstats.hub.callback;
+
+import lombok.SneakyThrows;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.glassfish.jersey.servlet.ServletContainer;
+
+import java.net.InetAddress;
+
+public class CallbackServer {
+
+    private static final int JETTY_PORT = 8090;
+    private Server jettyServer;
+
+    @SneakyThrows
+    public void start() {
+
+        final ServletContextHandler context = new ServletContextHandler(ServletContextHandler.SESSIONS);
+        context.setContextPath("/");
```

Comment:
If we want to end up running these tests concurrently, or even if we don't but less often, using a base path for all callbacks will result at some point in leakage between tests unfortunately. I'd like to see this at some point having a unique context path for each test, which could probably be injected at creation of the server. Another thing that doesn't have to be in this PR necessarily but worth mentioning